server/events/project_command_context_builder.go

```diff
@@ -37,7 +37,7 @@ type ProjectCommandContextBuilder interface {
 		prjCfg valid.MergedProjectCfg,
 		commentFlags []string,
 		repoDir string,
-		automerge, deleteSourceBranchOnMerge, parallelApply, parallelPlan, verbose bool,
+		automerge, deleteSourceBranchOnMerge, parallelApply, parallelPlan, verbose, force bool,
```

Comment:
I think these flags should be consolidated into a struct.

Reply:
done